triton-docker

```diff
@@ -71,6 +59,66 @@ function install () {
     echo
 }
 
+# Emit the current Triton profile and time
+function currentProfile () {
+    if [ -n "$TRITON_PROFILE" ]
+    then
+        local profile_source='via env var; use `eval "$(triton env <profile name>)"` to change'
+    else
+        local profile_source='default; use `triton profile set <profile name>` to change'
+    fi
+
+    echo "Executing in '$(triton profile get | awk 'NR == 1 {print $2}')' (${profile_source})"
+}
+
+# Set env vars to make using CNS easier
+function CnsEnvVars () {
+
+    # If the CNS env vars are already set, don't bother continuing
+    if [ -n "$TRITON_CNS_SEARCH_DOMAIN_PUBLIC" ] && [ -n "$TRITON_CNS_SEARCH_DOMAIN_PRIVATE" ]
+    then
+        return
+    fi
+
+    # Get and the user's account information and CloudAPI URL for parsing later
+    local triton_account="$(triton account get)"
+    local triton_url="$(triton profile get | awk -F"/" '/url:/{print $3}')"
+
+    # Do not continue if the target is not a Triton Public Cloud data center
+    if [ ! "api.joyent.com" == "$(echo "${triton_url}" | awk -F'.' '{print $2 "." $3 "." $4}')" ]
+    then
+        return
+    fi
+
+    # Check if CNS is enabled, require it
+    local triton_cns_enabled="$(echo "${triton_account}" | awk -F": " '/cns/{print $2}')"
+    if [ ! "true" == "$triton_cns_enabled" ]
+    then
+        echo
+        tput rev  # reverse
+        tput bold # bold
+        echo 'Error! Triton CNS is required and not enabled.'
```

Comment:
Well, if the reports are that CNS is now on by default are correct, then maybe not...